overblog/graphql-bundle/0.9/config/graphql/types/query.yaml

```diff
@@ -0,0 +1,8 @@
+Query:
+    type: object
+    config:
+        description: A Hello World query.
+        fields:
+            hello:
+                type: String
+                resolve: "world"
```

Comment:
Should probably be removed as this is an example.

Reply:
@fabpot removed, we only now point the user to the docs